tests/integration/golang/mlflow/artifact/list_test.go

```diff
@@ -40,7 +40,7 @@ func TestListArtifactTestSuite(t *testing.T) {
 }
 
 func (s *ListArtifactTestSuite) SetupTest() {
-	s3Client, err := helpers.NewS3Client(os.Getenv("S3_BASE_URL"))
+	s3Client, err := helpers.NewS3Client(os.Getenv("FML_S3_ENDPOINT_URI"))
```

Comment:
I think we should use a helper which defaults to `http://localhost:9000` for consistency

Reply:
fixed.